### DISCUSSION TITLE
Oracle Error following Tutorial: ORA-00907: missing right parenthesis

### DISCUSSION THREAD

**Discussion by DamolaGit** [Budibase/budibase#13997] [Help]
I am running into an error while following the steps of a Budi Base tutorial on YouTube.

(https://www.youtube.com/watch?v=5p4Vwv1xQl0&list=PLEel-MMkFaJnO9iy5IEA_tWnQzti6Hn0P)

I was following this tutorial step by step and once I got to the part where you set the filter for the Card block and use a binding, I'm always getting an error.  

![Filter Binding1](https://github.com/Budibase/budibase/assets/69820192/9c65b01f-39f2-4d20-8013-40773ee704de)

I am always getting this error [ORA-00907: missing right parenthesis] even when I try different layouts with a table for example, I am getting the same error

![Oracle Error](https://github.com/Budibase/budibase/assets/69820192/37331750-7681-479f-88f6-3224fdd62669)
 
I have checked the tables I imported and I can't find where the missing parenthesis is. Could there be something else within Budi base that I am doing wrong?

Please any help would be greatly appreciated.

**Comment by skarious** [comment]
I've have the same problem!

**Reply by mikesealey** [reply]
@skarious what version is your app using? Work was done to address this issue back in early August.
